nexus/db-queries/src/db/datastore/vpc.rs

```diff
@@ -740,6 +753,64 @@ impl DataStore {
             })
     }
 
+    pub async fn nexus_inbound_icmp_view(
+        &self,
+        opctx: &OpContext,
+    ) -> Result<ServiceIcmpConfig, Error> {
+        opctx.authorize(authz::Action::Read, &authz::FLEET).await?;
+        use nexus_db_schema::schema::vpc_firewall_rule::dsl;
+
+        let conn = self.pool_connection_authorized(opctx).await?;
+
+        let rule = dsl::vpc_firewall_rule
+            .filter(dsl::time_deleted.is_null())
+            .filter(dsl::vpc_id.eq(*SERVICES_VPC_ID))
+            .filter(dsl::name.eq(NEXUS_ICMP_FW_RULE_NAME.to_string()))
```

Comment:
I think that [`&'static str` implements `diesel::expression::AsExpression`](https://docs.diesel.rs/master/diesel/expression/trait.AsExpression.html#impl-AsExpression%3CNullable%3CText%3E%3E-for-%26str), so it shouldn't _really_ be necessary to `to_string()` this, AFAIK?
```suggestion
            .filter(dsl::name.eq(NEXUS_ICMP_FW_RULE_NAME))
```

Reply:
(sorry in advance if this turns out to be not compiley)